src/sentry/processing/realtime_metrics/__init__.py

```diff
@@ -20,8 +20,4 @@
     validate = __realtime_metrics_store__.validate
     record_project_duration = __realtime_metrics_store__.record_project_duration
     projects = __realtime_metrics_store__.projects
```

Comment:
I don’t think we need this anymore. It also does an expensive `scan` which we want to avoid.